files/zh-cn/web/javascript/reference/global_objects/symbol/matchall/index.md

```diff
@@ -14,7 +14,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Symbol/matchAll
 此 Symbol 用于 {{jsxref("String.prototype.matchAll()")}} 特别是 {{jsxref("RegExp.@@matchAll", "RegExp.prototype[@@matchAll]()")}}。下面两个例子返回相同的结果：
 
 ```js
-'abc'.matchAll(/a/);
+'abc'.matchAll(/a/g);
 
 /a/[Symbol.matchAll]('abc');
```

Comment:
Just update the code example as the one in english document:

https://github.com/mdn/content/blob/5a121576787f1fc93dc1f9eb31f74eaa47e629b8/files/en-us/web/javascript/reference/global_objects/symbol/matchall/index.md?plain=1#L33-L42

Reply:
Got it!